scalding-repl/src/main/scala/com/twitter/scalding/ScaldingILoop.scala

```diff
@@ -18,12 +18,16 @@ package com.twitter.scalding
 import java.io.File
 
 import scala.tools.nsc.interpreter.IR
+import scala.tools.nsc.GenericRunnerSettings
 
 /**
  * A class providing Scalding specific commands for inclusion in the Scalding REPL.
  */
 class ScaldingILoop
   extends ILoopCompat {
+
+  settings = new GenericRunnerSettings({ s => echo(s) })
```

Comment:
I think thats ok, echo should inform the user of any error, so that seems ok.


Can we access the parent settings without overriding them like this?